src/pyFAI/gui/pilx/MainWindow.py

```diff
@@ -141,6 +152,31 @@ def initData(self,
             else:
                 self._offset = 0
 
+            # treat source dataset ...
+            try:
+                image_grp = h5file[path]
+            except KeyError:
+                error_msg = f"Cannot access diffraction images at {path}: not such path."
+                logging.warning(error_msg)
+                status_bar = self.statusBar()
+                if status_bar:
+                    status_bar.showMessage(error_msg)
+            else:
```

Comment:
It is really worth continuing the execution of the function if the image group cannot be accessed ?

Reply:
There are plenty of `self.blabla` initialized later on. Putting a return there would prevent them from being initialized. I have no clue what would be the behaviour of the application. You are the author more than me. 
If you believe it is safe to return there, I see no objection.

Reply:
Let's leave it like that, I'll check later.

Reply:
Sometimes when I was writing and testing the widget, I moved the file or deleted the raw data, so there was no 2D pattern displayed but I still could test the map and the integrated pattern. I think the program should continue the execution, at least if we cannot see the data, we know immediately that there is a problem with the path of the data